libs/sdk-ui-kit/src/typings.d.ts

```diff
@@ -1,6 +1,7 @@
 // (C) 2020 GoodData Corporation
 declare module "react-native-listener";
 declare module "fixed-data-table-2";
+declare module "react-portal";
```

Comment:
There are "proper" types available at https://www.npmjs.com/package/@types/react-portal please try using them instead.

Reply:
On a quick glance they shouáld be compatible with our usage no problem.

Reply:
added `"@types/react-portal": "~4.0.2"`.